Documentation/kubernetes/install.rst

```diff
@@ -286,44 +286,174 @@ Deploying the DaemonSet
+
+If you are running more than one nodes simply add it to the list of endpoints.
+The list of endpoints can accept both domain names or IP addresses.
+Make sure you specify the correct port used in your etcd node.
+
+If etcd is running with TLS, there are a couple of changes that you need to do.
+
+#. Make sure you have ``https`` in all endpoints;
+
+#. Uncomment the line ``#ca-file: '/var/lib/etcd-secrets/etcd-ca'`` so that the
+   certificate authority of the servers are known to Cilium;
+
+#. Create a kubernetes secret with certificate authority file in kubernetes;
+
+    #. Create a file with the name ``ca.crt`` that contains the certificate
+       authority inside of it;
+
+    #. Create the secret by executing:
+
+        .. code-block:: bash
+
+            $ kubectl create secret generic -n kube-system cilium-etcd-secrets \
+                --from-file=etcd-ca=ca.crt
+
+
+If etcd is running with client to server authentication, you need make more
+changes to the ConfigMap
+
+#. Uncomment both lines ``#key-file: '/var/lib/etcd-secrets/etcd-client-key'``
+   and ``#cert-file: '/var/lib/etcd-secrets/etcd-client-crt'``;
+
+#. Create a kubernetes secret with certificate authority file in kubernetes.
+
+    #. Create a file with the name ``client.key`` that contains the client key
+       inside of it;
+
+    #. Create a file with the name ``client.crt`` that contains the client
+       certificate inside of it;
+
+    #. Create the secret by executing:
+
+        .. code-block:: bash
+
+            $ kubectl create secret generic -n kube-system cilium-etcd-secrets \
+                --from-file=etcd-ca=ca.crt \
+                --from-file=etcd-client-key=client.key \
+                --from-file=etcd-client-crt=client.crt
+
+
+.. note::
+
+    If you have set up the secret you might see the error
+    ``Error from server (AlreadyExists): secrets "cilium-etcd-secrets" already exists``
+    you can simply delete it with
+    ``kubectl delete secret -n kube-system cilium-etcd-secrets``
+    and re-create it again.
+
+
+.. note::
+
+    When creating the kubernetes secrets just make sure you create it with
+    all necessary files, ``ca.crt``, ``client.key`` and ``client.crt`` in a
+    single ``kubectl create``.
+
+Regarding the etcd configuration that is all you need to change in the
+`ConfigMap`.
+
+Adjusting Cilium Options
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the `ConfigMap` there are a couple of options that can be changed
+accordingly with your changes.
+
+* ``debug`` - Sets to run Cilium in full debug mode, it can be changed at
+  runtime;
+
+* ``disable-ipv4`` - Disables IPv4 in Cilium and endpoints managed by Cilium;
+
+* ``sidecar-http-proxy`` - Runs the HTTP proxy as a sidecar;
+
+* ``clean-cilium-state`` - Removes any Cilium state, e.g. BPF policy maps,
+  before starting the Cilium agent;
+
+* ``legacy-host-allows-world`` - If true, the policy with the entity
+  ``reserverd:host`` allows traffic from ``world``. If false, the policy needs
```

Comment:
`reserved:host` typo. I also saw this below.

Also, the default is true. I think it's useful to know what the default is when it's not specified.